microsoft-edge/webview2/roadmap.md

```diff
@@ -59,4 +59,4 @@ You can access WebView2 controls in your applications using [Windows UI Library
 
 #### Xbox, HoloLens, and XAML limitations
 
-This version of WebView 2 will only work on PC-class devices, offering the full range of capability found in the WinUI 3 variant. Xbox, HoloLens, and XAML Island support requires additional work, and these devices and scenarios may be considered at a future point in time.
+This version of WebView 2 will only work on PC-class devices, offering the full range of capability found in the WinUI 3 variant.<!--todo: reword--> Xbox, HoloLens, and XAML Island support requires additional work, and these devices and scenarios may be considered at a future point in time.
```

Comment:
Let's reword this section to:
Title: XAML Limitation
XAML Island support requires additional work and may be considered for future releases.

Reply:
fixed locally
```
#### XAML limitation

XAML Island support requires additional work and may be considered for future releases.
```